src/index.ts

```diff
@@ -269,6 +269,16 @@ process.on("SIGUSR2", async () => await cleanupJobResources(jobs));
             description: "The name, without the architecture, of a gitlab hosted runner to emulate. See here: https://docs.gitlab.com/ee/ci/runners/hosted_runners/linux.html#machine-types-available-for-linux---x86-64",
             choices: GitlabRunnerPresetValues,
         })
+        .option("color", {
+            requiresArg: false,
+            default: false,
+            // placeholder for the --no-color option
+            hidden: true,
+        })
+        .option("no-color", {
```

Comment:
```suggestion
        // NOTE: not redundant, this is added for the `--help` message
        .option("no-color", {
```

Reply:
It is redundant until we decide to properly add `--no` versions for every boolean values in index.ts

![image](https://github.com/user-attachments/assets/f373548c-6638-49df-9c40-394a811a104c)